build.py

```diff
@@ -68,16 +68,15 @@
 # different versions are used then one backend or the other will
 # incorrectly load the other version of the openvino libraries.
 #
-TRITON_VERSION_MAP = {
-    "2.49.0dev": (
-        "24.08dev",  # triton container
-        "24.07",  # upstream container
-        "1.18.1",  # ORT
-        "2024.0.0",  # ORT OpenVINO
-        "2024.0.0",  # Standalone OpenVINO
-        "3.2.6",  # DCGM version
-        "0.5.3.post1",  # vLLM version
-    )
+DEFAULT_TRITON_VERSION_MAP = {
```

Comment:
Willing to understand motivation behind this utility better also wondering if it pass all other backends?

Reply:
While updating code I found it very hard to refer to indexes. Prefer using key names to retrieve values instead of indices.
Please let me know how to verify other backend builds / any other verifications I should do here

Reply:
You need to start the general purpose pipeline with all enabled flags against current branch.

Reply:
thanks, started 17226628

Reply:
all build steps are passing on this pipeline

Reply:
If the idea is to give users more control against the versions. 
May be it make sense to provide such a values as an variables?

Reply:
users can now pass values for any of these variables, it still makes sense to keep the default dictionary

Reply:
Our end users are developers and most like behavior for them is to clone and change the code, suppose it's overhead.

Reply:
Was an agreement reached on the overhead concern?

Reply:
@mc-nv Do you still have concern over the changes?